plugins/smartvol/rest.go

```diff
@@ -28,11 +28,11 @@ func validateVolCreateReq(req *smartvolapi.VolCreateReq) error {
 	}
 
 	if req.Transport != "" && req.Transport != "tcp" && req.Transport != "rdma" {
-		return errors.New("Invalid transport. Supported values: tcp or rdma")
+		return errors.New("invalid transport. Supported values: tcp or rdma")
 	}
 
 	if req.Size < minVolumeSize {
-		return errors.New("Invalid Volume Size, Minimum size required is " + strconv.Itoa(minVolumeSize))
+		return errors.New("invalid Volume Size, Minimum size required is " + strconv.Itoa(minVolumeSize))
```

Comment:
Volume Size -> volume size and a full-stop instead of comma. Can be addressed in another PR